README.md

```diff
@@ -0,0 +1,97 @@
+# Juvix
+
+<a href="https://github.com/anoma/juvix/actions/workflows/ci.yml">
+<img alt="CI status" src="https://github.com/anoma/juvix/actions/workflows/ci.yml/badge.svg" />
+</a>
+
+<a href="https://github.com/anoma/juvix/actions/workflows/pages/pages-build-deployment"><img
+src="https://github.com/anoma/juvix/actions/workflows/pages/pages-build-deployment/badge.svg"
+alt="pages-build-deployment" /></a>
+
+<a href="https://github.com/anoma/juvix/tags">
+<img alt="" src="https://img.shields.io/github/v/release/anoma/juvix?include_prereleases" />
+</a>
+
+<a href="https://github.com/anoma/juvix/blob/main/LICENSE">
+<img alt="LICENSE" src="https://img.shields.io/badge/license-GPL--3.0--only-blue.svg" />
+</a>
+
+<a href="https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=455254004">
+<img height="20pt" alt="Open the Juvix Standard Lib in Github Codespace" src="https://github.com/codespaces/badge.svg" />
+</a>
+
+<a href="https://github.com/anoma/juvix">
+<img align="right" width="270" alt="Juvix Mascot" src="assets/images/tara-seating.svg" />
+</a>
+
+Juvix is an open-source, constantly evolving functional programming language
+designed for writing privacy-preserving decentralized applications. It enables
+developers to write smart contracts and decentralized applications on blockchain
+platforms in a secure and efficient way.
+
+## Getting Started
+
+To get started with Juvix, head over to the [documentation website](https://docs.juvix.org) to learn more about the language and its features. You can also find installation instructions and tutorials to help you get started with writing Juvix programs.
+You can download the [latest release](https://github.com/Juvix-lang/juvix/releases) from the Juvix GitHub repository or use the web-based development environment, [Juvix Github
+Codespace](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=102404734&machine=standardLinux32gb&location=WestEurope), which provides a pre-configured workspace with Juvix and the Haskell toolchain installed and ready to use.
+
+## The Juvix programming language
+
+Juvix provides a high degree of assurance. The Juvix compiler runs
+several static analyses which guarantee the absence of runtime errors.
+Analyses performed include termination, arity, and type checking. As a
+result, functional programs, especially validity predicates, can be
+written with greater confidence in their correctness.
+
+Some language features in Juvix include:
+
+- unicode syntax
+- parametric polymorphism
+- inductive and parametric data types
+- higher-order functions
+- implicit arguments
```

Comment:
While we have support for implicit arguments syntax, maybe it would better reflect the current state of the language to say "type inference". I would probably remove "holes in expressions" too, since these can only be types and that is included in type inference.

Reply:
Done.